plugins/http_plugin/tests/unit_tests.cpp

```diff
@@ -646,9 +652,13 @@ BOOST_FIXTURE_TEST_CASE(bytes_in_flight, http_plugin_test_fixture) {
    std::this_thread::sleep_for(std::chrono::seconds(1));
```

Comment:
Can we get rid of this final sleep? It seems like waiting for each connection to indicate it's readable would suffice. But if we need to make another busy loop maybe we could wait to see requests_in_flight is 32